lib/services/ios-log-parser-service.ts

```diff
@@ -41,12 +41,12 @@ export class IOSLogParserService extends EventEmitter implements IIOSLogParserSe
 		}
 	}
 
-	private startLogProcess(device: Mobile.IDevice): void {
+	private startLogProcess(device: Mobile.IDevice): Promise<void> {
 		if (device.isEmulator) {
 			return this.$iOSSimulatorLogProvider.startNewMutedLogProcess(device.deviceInfo.identifier);
 		}
 
-		return this.$iosDeviceOperations.startDeviceLog(device.deviceInfo.identifier);
+		return Promise.resolve(this.$iosDeviceOperations.startDeviceLog(device.deviceInfo.identifier));
```

Comment:
you can make the method async and just return the `this.$iosDeviceOperations.startDeviceLog(device.deviceInfo.identifier)` call here.